homeassistant/components/cloud/account_link.py

```diff
@@ -18,6 +18,9 @@
 _LOGGER = logging.getLogger(__name__)
 
 CURRENT_VERSION = AwesomeVersion(HA_VERSION)
+CURRENT_PLAIN_VERSION = AwesomeVersion(
+    CURRENT_VERSION.string.removesuffix(f".{CURRENT_VERSION.modifier}")
```

Comment:
```suggestion
    CURRENT_VERSION.string.removesuffix(f"{CURRENT_VERSION.modifier}")
```

Our beta versions are `0bX` and not `0.bX`

Reply:
Nicely caught, pushed a change.